corehq/apps/accounting/payment_handlers.py

```diff
@@ -409,14 +409,15 @@ def _pay_invoice(self, invoice):
                     autopay_card,
                     amount_in_dollars=amount,
                     description='Auto-payment for Invoice %s' % invoice.invoice_number,
+                    idempotency_key=invoice.id
```

Comment:
Mainly verifying my understanding here. When we had looked at this, you mentioned the risk of using the invoice.id as a unique identifier on the PaymentRecord because of the possibility of customers paying for the same invoice in multiple installments. However I think that was only a non-autopay issue, so this solution is ideal since it only impacts autopay. Is that right?

Reply:
Yes. That's correct.